spp_api/controllers/api.py

```diff
@@ -130,6 +151,20 @@ def read_multi__GET(self, namespace, version, model, **kw):
             "limit": kw.get("limit", 0),
             "version": version,
         }
+
+        # Request Log
+        self.create_api_log(
```

Comment:
I think we should implement this by creating a decorator for all the `create_api_log`

Reply:
@nhatnm0612 sure, let me try that. Thanks for the suggestion.